config/galaxy.ini

```diff
@@ -1,1160 +0,0 @@
-#
```

Comment:
I can't see to see this in my repo at all so not sure how to revert/delete it! Argh!